web/nicolleta1981/html/Value attribute/value attribute.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <title>Value attribute</title>
+  </head>
+
+  <body>
+    <h1>The HTTP method that the browser uses to submit the form can have values bellow:
+      <ul>
+        <li>post-HTTP POST method-form data are included in the body of the form and sent to the server</li>
+        <li>get-HTTP GET method-form data are appended to the action attribute URI with a '?'as separator, and the resulting URI is sent to the server. Use this method when the form has no side-effects and contains only ASCII characters</li>
+        <li>dialog-Use when the form is inside a < dialog > element to close the dialog when submitted.
+          <p>This value can be overriden by a formmethod attribute on a < button >or < input >element.</p>
```

Comment:
Button must be closed or deleted here ,on input you want to put atleast a "type" attribute